src/cmd.rs

```diff
@@ -441,7 +444,7 @@ impl<'a> Parse<'a> for Command {
             }
             "source" => {
                 let (path, p) = p.word()?;
```

Comment:
Almost there! What I meant is that here you would say `p.path()?` instead of `p.word()?`

Reply:
.. and do the expansion in there.

Reply:
I have since done so.